cvat/apps/git/tests.py

```diff
@@ -5,3 +5,59 @@
 from django.test import TestCase
 
 # Create your tests here.
+
+from cvat.apps.git.git import Git
```

Comment:
Will it be run with all other tests?

Reply:
Now it should, as well as other tests besides `apps.engine`, if they existed.